prober/utils.go

```diff
@@ -135,6 +135,6 @@ func chooseProtocol(ctx context.Context, IPProtocol string, fallbackIPProtocol b
 
 func ipHash(ip net.IP) float64 {
 	h := fnv.New32a()
-	h.Write(ip)
+	h.Write([]byte(ip.String()))
```

Comment:
I did some testing, I can reproduce this issue. The fix is this:

```suggestion
	if ip.To4() != nil {
		h.Write(ip.To4())
	} else {
		h.Write(ip.To16())
	}
```